src/System.Windows.Forms.Design/src/System/Windows/Forms/Design/ControlDesigner.cs

```diff
@@ -85,21 +83,21 @@ private Point Location
         {
             Point loc = Control.Location;
 
-            ScrollableControl p = Control.Parent as ScrollableControl;
-            if (p is not null)
+            ScrollableControl? parent = Control.Parent as ScrollableControl;
+            if (parent is not null)
```

Comment:
Let's change to use pattern matching here. Ditto with the setter as well.
```suggestion
            if (Control.Parent is ScrollableControl parent)
```